### ISSUE TITLE
PSCI test failures on Windows

### ISSUE THREAD

**Issue by kRITZCREEK** [purescript/purescript#2881]
Not really sure what's causing this, but when running the testsuite on a fresh Windows install I get these failures:

```
evalTests
(line 1, column 1):
unexpected end of input
expecting "import", declaration, expression or "let"
  Multiline.purs FAILED [1]
(line 1, column 1):
unexpected end of input
expecting "import", declaration, expression or "let"
  BasicEval.purs FAILED [2]

Failures:

  tests\TestPsci\EvalTest.hs:55:
  1) evalTests Multiline.purs
       uncaught exception: ExitCode (ExitFailure 1)

  tests\TestPsci\EvalTest.hs:55:
  2) evalTests BasicEval.purs
       uncaught exception: ExitCode (ExitFailure 1)
```

They work fine on my other Windows machine, so I'm really not sure what's going on but maybe @noraesae or @hdgarrood have an idea?

One difference is that this machine is running node 7.10, so that might be related.

I tried it with both `cmd.exe` and powershell and got the same result

**Comment by noraesae** [comment]
It says `unexpected end of input ` at `(line 1, column 1)`, so it seems it's failing to read the files themselves. I am not familiar with Windows, but could there be permission problem?

**Comment by vendethiel** [comment]
Is it in any way possible these lines have a BOM?

**Comment by kRITZCREEK** [comment]
@noraesae I can open them in the same shell as I run the tests in, so I'd think the permission are fine?

```
λ  more .\examples\psci\Multiline.purs             
import Prelude                                     
import Data.Array                                  
                                                   
-- @paste                                          
fac :: Int -> Int                                  
fac n = foldl mul 1 (1..n)                         
-- @paste                                          
                                                   
-- @shouldEvaluateTo 3628800                       
fac 10                                                                                             
```
@vendethiel How would I go about checking that and how would it matter?

**Comment by kRITZCREEK** [comment]
Allright so I figured it out. The problem is basically the same that I was facing in #2876. Because `readFile` from ByteString does not turn `CRLF` into `LF` on Windows, the result of our `System.IO.UTF8.readUTF8File` doesn't work properly in combination with the `lines` function.

It's a shame that there is no way to read a UTF8 encoded *text* file in either `bytestring` or `text`. What I ended up doing is replacing `\r\n` with `\n` manually, so that might be the simplest fix here?
